core/src/main/java/org/wildfly/extras/sunstone/api/impl/ObjectProperties.java

```diff
@@ -26,12 +26,19 @@
 /**
  * Object which holds an {@link ObjectPropertiesType}, a name and object properties.
  *
+ * TODO: better documentation for this class, there are many public methods which are not intuitive and may not be meant to used by public
+ * TODO: e.g. applyOverrides, applyMissing, getArray
  */
 public class ObjectProperties implements ConfigProperties {
     private static final Logger LOGGER = SunstoneCoreLogger.DEFAULT;
 
     public static final String PROPERTIES_FILE_KEY = "properties";
+    /**
+     * Replaced by {@link ObjectProperties#SYSTEM_PROPERTY_VALUE_DELIMITER_SST}.
+     */
+    @Deprecated
     public static final String SYSTEM_PROPERTY_VALUE_DELIMITER = "clouds.sysprop.value.delimiter";
+    public static final String SYSTEM_PROPERTY_VALUE_DELIMITER_SST = "sunstone.sysprop.value.delimiter";
```

Comment:
What is the _SST here? Just for clarification for me.

Reply:
That's a short name for Sunstone.

Reply:
If you're fine with abbreviation, I'm fine too, I just don't see the point in abbreviating for already long name. :)